src/firefly/js/templates/fireflyviewer/LandingPage.jsx

```diff
@@ -65,15 +67,15 @@ export function LandingPage({slotProps={}, sx, ...props}) {
 
 function DefaultAppBranding({appTitle, appDescription}) {
     return (
-        <Stack spacing={.25}>
-            <Typography level={'h4'}>{`Welcome to ${appTitle}`}</Typography>
+        <Stack spacing={.25} alignItems='center'>
+            <Typography level={'h1'}>{`Welcome to ${appTitle}`}</Typography>
```

Comment:
Default h1 font style is too strong in font weight and color. I get that scientists want it to be like Rubin but that styling is different:
https://github.com/lsst/suit/blob/main/src/suit/js/RubinLanding.jsx#L9

Reply:
thanks, I will change it